test/integration/CurveTricryptoPairTest.t.sol

```diff
@@ -10,6 +10,7 @@ import {Settler} from "src/Settler.sol";
 import {SettlerBase} from "src/SettlerBase.sol";
 import {ISettlerActions} from "src/ISettlerActions.sol";
 import {ISignatureTransfer} from "@permit2/interfaces/ISignatureTransfer.sol";
+import {ISettlerBase} from "src/interfaces/ISettlerBase.sol";
```

Comment:
This was added but `SettlerBase` import was not removed. It can be removed in most of the modified files inside `test/integration`